src/__tests__/events.js

```diff
@@ -137,6 +137,12 @@ const eventTypes = [
   },
 ]
 
+const allEvents = eventTypes
```

Comment:
I'm happy to revert this to be more straightforward, but it was getting a bit verbose/has potential for drift.